distribution/lib/Standard/Snowflake/0.0.0-dev/src/Internal/Snowflake_Dialect.enso

```diff
@@ -552,6 +553,37 @@ make_order_descriptor internal_column sort_direction text_ordering =
                             folded_expression = SQL_Expression.Operation "LOWER" [upper]
                             Order_Descriptor.Value folded_expression sort_direction nulls_order=nulls collation=Nothing
 
+## PRIVATE
+round_bankers = Base_Generator.lift_binary_op "ROUND_BANKERS" x-> dp->
```

Comment:
no way to use the built in bankers rounding (via some casting,,,)?

Reply:
Snowflake's builtin banker's rounding is only for fixpoint. If we cast from float to fixpoint, we would have to restrict to the values that are representable in fixpoint; for all others, we'd have to convince ourselves that they would not be affected by the rounding anyway, or would be rounded to 0, and I'm not sure that would work.